docs/api-usage/i18n.mdx

```diff
@@ -0,0 +1,88 @@
+- Localizing display formats of **prices**, **dates**, and other data in the storefront.
+- Localizing **address** input.
+- Localizing **payment** and **shipping** methods.
+
+## Translating the storefront application
+
+For web applications, you can manage the frontend part using libraries like [Format.JS](https://formatjs.io). Generated files can be later edited manually or using an external service, for example, [Transifex](https://www.transifex.com) or [POEditor](https://poeditor.com).
+
+## Translating the product catalog
+
+For translating database data, Saleor delivers specialized translation objects which can be modified in the dashboard or using the API.
+
+- [`ProductTranslation`](api-reference/objects/product-translation.mdx)
+- [`CollectionTranslation`](api-reference/objects/collection-translation.mdx)
+- [`CategoryTranslation`](api-reference/objects/category-translation.mdx)
+- [`AttributeTranslation`](api-reference/objects/attribute-translation.mdx)
+- [`AttributeValueTranslation`](api-reference/objects/attribute-value-translation.mdx)
+- [`ProductVariantTranslation`](api-reference/objects/product-variant-translation.mdx)
+- [`PageTranslation`](api-reference/objects/page-translation.mdx)
+- [`ShippingMethodTranslation`](api-reference/objects/shipping-method-translation.mdx)
+- [`SaleTranslation`](api-reference/objects/sale-translation.mdx)
+- [`VoucherTranslation`](api-reference/objects/voucher-translation.mdx)
+- [`MenuItemTranslation`](api-reference/objects/menu-item-translation.mdx)
+
+### Fetching translated data
+
+Here's an example fetching the Polish translation of an existing category. A `translation` field in the [`Category`](api-reference/objects/category.mdx) model takes a `languageCode` argument with one of the values defined by [`LanguageCodeEnum`](api-reference/enums/language-code-enum.mdx):
+
+```graphql
+query {
+  category(id: "Q2F0ZWdvcnk6OQ==") {
+    name
+    seoDescription
+    // highlight-next-line
+    translation(languageCode: PL) {
+      name
+      seoDescription
+    }
+  }
+}
+```
+
+Response:
+
+```json
+{
+  "data": {
+    "category": {
+      "name": "Shoes",
+      "seoDescription": "They go on your feet.",
+      "translation": {
+        "name": "Buty",
+        "seoDescription": null
+      }
+    }
+  }
+}
+```
+
+Fields that haven't been translated will return `null`.
+
+## Localizing prices and dates
+
+Prices and dates can be localized using [`Intl.NumberFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat) and [date-fns](https://date-fns.org).
+
+## Localizing address input
+
+Saleor provides an [`AddressInput`](api-reference/inputs/address-input.mdx) type that can be used to collect address data from the user. It contains fields for the country, city, postal code, and street address. Which fields should be displayed to the user and how to label them can be determined by querying the address validation API.
```

Comment:
```suggestion
Saleor provides an [`AddressInput`](api-reference/inputs/address-input.mdx) type that can be used to collect address data from the user. It contains country, city, postal code, and street address fields. Which fields should be displayed to the user and how to label them can be determined by querying the address validation API.
```

Reply:
I think it's easier to read this the way it was before. In your version, I have to get through the entire litany of names before I get to "fields" which is the whole point of this sentence.